index.js

```diff
@@ -2,10 +2,8 @@ const core = require("@actions/core");
 const exec = require("@actions/exec");
 const { promises: fs } = require("fs");
 
-const host =
-  core.getInput("owner") === "Staging-ObelusFamily"
-    ? "https://engine-staging.wilco.gg"
-    : "https://engine.wilco.gg";
+const host = 
+  core.getInput("engine") || (core.getInput("owner") === "Staging-ObelusFamily" ? "https://engine-staging.wilco.gg" : "https://engine.wilco.gg");
```

Comment:
What's the value of `core.getInput("engine")` It's the env var?

Reply:
yes, see here: https://github.com/trywilco/wilco-engine/pull/1553